src/component/Header/Header.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Header.scss';
+import './HeaderResponsive.scss';
+
+function Header() {
+    return (
+        <>
+            <div id="main-header" className="main-header">

```

Comment:
always use header tag with header id for website's header structure like header#header. and don't use class for main header of the website.